packages/analytics-js-integrations/src/integrations/GoogleAds/browser.js

```diff
@@ -45,6 +46,7 @@ class GoogleAds {
     // Depreciating: Added to make changes backward compatible
     this.dynamicRemarketing = config.dynamicRemarketing;
     this.allowEnhancedConversions = config.allowEnhancedConversions || false;
+    this.v2 = config.v2 || true;
```

Comment:
Default should be `false` right? To backward compatibility

Reply:
No, for new destinations created going ahead we want to keep it enabled, so that ecom spec is followed.

For existing destinations to avoid any breaking change, we will have a migration to disable it